StellaNowSDKDemo/src/Program.cs

```diff
@@ -74,6 +82,8 @@ private async Task RunAsync()
             
             // Delay between messages
             await Task.Delay(TimeSpan.FromSeconds(1));
```

Comment:
Why hardcoding 1 second ?

Reply:
Its just a demo app, not part of the SDK itself